test/lvc_SUITE.erl

```diff
@@ -14,15 +14,63 @@
 %% Copyright (c) 2007-2012 VMware, Inc.  All rights reserved.
 %%
 
--module(rabbit_lvc_test).
+-module(lvc_SUITE).
 
 -include_lib("eunit/include/eunit.hrl").
+-include_lib("common_test/include/ct.hrl").
 -include_lib("amqp_client/include/amqp_client.hrl").
 
-lvc_test() ->
-    {ok, Conn} = amqp_connection:start(#amqp_params_network{}),
-    {ok, Ch} = amqp_connection:open_channel(
-                 Conn, {amqp_direct_consumer, [self()]}),
+-compile(export_all).
+
+all() ->
+    [
+     {group, non_parallel_tests}
+    ].
+
+groups() ->
+    [
+     {non_parallel_tests, [], [
+                               lvc
+                              ]}
+    ].
+
+%% -------------------------------------------------------------------
+%% Testsuite setup/teardown.
+%% -------------------------------------------------------------------
+
+init_per_suite(Config) ->
+    rabbit_ct_helpers:log_environment(),
+    inets:start(),
+    Config.
+
+end_per_suite(Config) ->
+    Config.
+
+init_per_group(_, Config) ->
+    Config1 = rabbit_ct_helpers:set_config(Config, [
+                                                    {rmq_nodename_suffix, ?MODULE}
```

Comment:
Only use `?MODULE` as a nodename suffix when the node is global to the testsuite. When nodes are started at the test group or testcase level, construct a suffix based on the group/testcase name.

Here is an example for a group-level node:
https://github.com/rabbitmq/rabbitmq-amqp1.0/blob/master/test/system_SUITE.erl#L63-L67